source/optional_features/prefill_user_details/index.html.md.erb

```diff
@@ -31,6 +31,13 @@ For example:
 
 All the parameters in `billing_address` are optional.
 
+Each value must be no longer than:
+
+- 254 characters for `email`
+- 255 characters for `cardholder_name`
+- 255 characters for `line1`, `line2` and `city` in `billing_address`
```

Comment:
I said something like this in a response to a support ticket and the user came back and asked if it meant 255 for each property or 255 characters in total across all those properties. Can we make it clear it’s per-field (which might be as simple as giving each of the four properties its own bullet point)?

Reply:
Good point. How about:
`line1`, `line2` ~~and~~or `city`

I think giving each item its own bullet would unfortunately lead to too much duplication of '255'.

Reply:
I suppose “255 characters each for `line1`, `line2` and `city` in `billing_address`” would work.

Reply:
Good plan! Added and tweaked slightly in https://github.com/alphagov/pay-tech-docs/pull/300/commits/d1ea6c6b7e6128ccb832c8fcf6cbf613ca5c09cf.